lib/fine-types/src/Language/FineTypes/Export/OpenAPI/Value/FromJSON.hs

```diff
@@ -0,0 +1,153 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Language.FineTypes.Export.OpenAPI.Value.FromJSON
+    ( jsonToValue
```

Comment:
```suggestion
    ( valueFromJson
```

I'm beginning to prefer the `xFromY` form for all directions, because it speaks more: In the expression `(xFromY ysomething)` "is an x", and `ysomething` "is a Y".